test/src/main/res/layout/activity_spinner_enable_from_xml.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2019 CondorLabs
```

Comment:
you should rename the file to activity_spinner_disable because the field of the component is disable